charts/humio-operator/templates/operator-rbac.yaml

```diff
@@ -1,16 +1,13 @@
 {{- if .Values.operator.rbac.create -}}
+{{- $commonLabels := include "humio.labels" . }}
```

Comment:
Needed because we change the scope with `range` later on (such as in line 12 bellow)